source/release-notes.txt

```diff
@@ -11,6 +11,46 @@ Release Notes
    :class: twocols
 
 
+1.10.0 Changelog
+---------------
+
+*Released October 1, 2024*
+
+New features:
+
+- New bulk conversion UI and API for Query Converter to automatically 
+  convert large numbers of SQL queries, views and stored procedures. 
+  See :ref:`rm-bulk-conversion`.
+- Major performance improvements with up to 5x faster new document 
+  creation. By default, snapshot jobs now create top-level documents 
+  using an improved, non-idempotent bulk insert method.
+- Added explicit control over idempotency behaviour through the 
+  ``migrator.engine.transform.idempotency: true`` setting in the 
+  ``user.properties`` file.
+- Updated SQL Server JDBC driver to the latest version.
+- Added Basic Authentication support for Kafka.
```

Comment:
(question ... or somethig)
I'm pretty sure Basic Authentication doesn't need to be a proper noun, right?

Reply:
I'm going to tag @tomhollander for input on this one. I believe in this context, "Basic" might be the only proper noun if we're going by the RFC. I've also seen "Basic HTTP authentication" thrown around.

Reply:
Taggin @davidsoulmdb for this one as well.

Reply:
My bad. I googled and see that Basic Authentication is sometimes used as a proper noun.